src/models/contracts.ts

```diff
@@ -70,7 +70,7 @@ export class SaveResultsRequestParams {
 export class SaveResultsAsCsvRequestParams extends SaveResultsRequestParams {
     includeHeaders: boolean = true;
     delimiter: string = ',';
-    lineSeperator: string = undefined;
+    lineSeparator: string = undefined;
```

Comment:
I don't think we can just change this here.  We'd also need to update the SQL Tools Service code that corresponds to this RPC method since presumably the same typo exists there. i.e., https://github.com/microsoft/sqltoolsservice/blob/98018c5292b3e8e91f0e055a094ca754abaa650e/src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/SaveResultsRequest.cs#L89.  And Azure Data Studio will also require this fix to stay in sync.

Reply:
Ahh, yeah, I was wondering about that. Unfortunate but I totally understand. There are likely other extensions that might have the same issue. Is there an easy way to remove that commit from this PR so the other change stands alone?